pkgs/by-name/do/double-entry-generator/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "double-entry-generator";
+  version = "2.7.1";
+  src = fetchFromGitHub {
+    owner = "deb-sig";
+    repo = "double-entry-generator";
+    hash = "sha256-2Y8Spj1LAVZsUgChDYDCZ63pTH+nqs2ff9xcmC+gr0c=";
+    rev = "v${version}";
+  };
+
+  vendorHash = "sha256-Xedva9oGteOnv3rP4Wo3sOHIPyuy2TYwkZV2BAuxY4M=";
+
+  postInstall = ''
+    rm $out/bin/hack
+  '';
+
+  meta = with lib; {
```

```suggestion
  passthru.updateScript = nix-update-script { };

  meta = with lib; {
```

Adding an update script makes @r-ryantm creates update PRs for you and ping you to test, verify and even merge.